### ISSUE TITLE
Users are unable to download files to device in Status

### ISSUE THREAD

**Issue by mjdietzx** [status-im/status-react#6435]
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a developer working on a DApp compatible with Status

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: 

To on-board our users they need to download a file. I've tried a few ways to allow them to download files, but best way seems to be: https://www.w3schools.com/tags/att_a_download.asp

All of these fail in Status with message:
```
Unable to load page
102
Frame load interrupted
```

#### Expected behavior
[comment]: # (Describe what you expected to happen.)

Users are able to download files to their device via status

#### Actual behavior
[comment]: # (Describe what actually happened.)

Downloads fail

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

When testing, add an a element with download attribute (ie an image) and see the fialure in Status

- Open Status
- ...
- Step 3, etc.

### Solution
[comment]: # (Please summarise the solution and provide a task list on what needs to be fixed.)
*Summary*: 

- [ ] ...
- [ ] ...

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: ...
[comment]: # (Android or iOS.)
* Operating System: iOS

#### Logs

[comment]: # (if on Android please replicate bug whilst running adb logcat)
```
...
```

<blockquote><div><strong><a href="https://www.w3schools.com/tags/att_a_download.asp">HTML a download Attribute</a></strong></div><div>Well organized and easy to understand Web building tutorials with lots of examples of how to use HTML, CSS, JavaScript, SQL, PHP, and XML.</div></blockquote>

**Comment by rachelhamlin** [comment]
Thanks for reporting this @mjdietzx! Can you share the URL to your DApp? I'd like to test this particular flow, too.

**Comment by mjdietzx** [comment]
hey @rachelhamlin, thanks for the response.

you can try out our DApp at https://blockimmo.ch

To actually see this bug, you will need to login (easy in Status - in browser requires MetaMask), and then go to your profile. After completing the first step of our user onboarding process (anti money laundering), you will be prompted to download our terms and conditions. This is where you will see the bug as downloads fail.

Basically after downloading our terms and conditions, the user see's this screen where they sign the agreement with their wallet (but we can't get here in Status). every other aspect of our DApp works in status except this specifc download
<img width="1279" alt="screen shot 2018-10-25 at 1 54 53 pm" src="https://user-images.githubusercontent.com/7217256/47498461-a238f500-d85d-11e8-8d00-61a0bd195e5f.png">

Thanks,
Mike

<blockquote><img src="/favicon.ico" width="48" align="right"><div><strong><a href="https://blockimmo.ch/">blockimmo | Real Estate Accessible For All</a></strong></div><div>A blockchain powered platform, our ecosystem enables fractionalized property investments and ownership. Fees are cut and real estate investments become liquid and manageable.</div></blockquote>

**Comment by rachelhamlin** [comment]
Hey again @mjdietzx - just to update you, we're looking into this issue. I've received your submission to have blockimmo listed in Status and the DApp looks great. We think it's probably best not to list the DApp until this bug is fixed, as it blocks a key step in onboarding. Am I understanding that correctly?

**Comment by mjdietzx** [comment]
Hey @rachelhamlin , thanks!

I've looked into this more, and I don't think this is actually a bug. What I realized is all browsers have this problem on iOS (safari + other DApp browsers). It is due to iOS being strict about downloading files onto the device, and not Status itself.

So this is a non-issue (I forgot to close this bug). Thanks for the help though!

And we have implemented this part of on-boarding differently so it works on mobile. So the DApp is fully functional now.
